packages/pnet/src/index.ts

```diff
@@ -81,7 +80,7 @@ class PreSharedKeyConnectionProtector implements ConnectionProtector {
     }
 
     if (connection == null) {
-      throw new CodeError(Errors.NO_HANDSHAKE_CONNECTION, codes.ERR_INVALID_PARAMETERS)
+      throw new CodeError(Errors.NO_HANDSHAKE_CONNECTION, Errors.ERR_INVALID_PARAMETERS)
```

Comment:
nit: we should normalize on `errors|codes.ACTUAL_ERR` and always import a singular object or always do `ACTUAL_ERR` 

also(prob for outside this PR but..), where does CodeError come from, and can we enforce the error types there? if we did, we could use generics on modules and ensure each module has their possible error types enforced too (makes it easier for devs to know what errors can be thrown)

Reply:
> where does CodeError come from

```
import { CodeError } from '@libp2p/interface/errors'
```

> we could use generics

Maybe, I think this might get quite unwieldy.  Feel free to open a PR to discuss?